src/libraries/System.Drawing.Common/tests/mono/System.Drawing/BitmapTests.cs

```diff
@@ -1219,78 +1219,6 @@ public void XmlSerialization()
             new XmlSerializer(typeof(Bitmap));
         }
 
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapImageCtor()
-        {
-            Assert.Throws<NullReferenceException>(() => new Bitmap((Image)null));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapImageSizeCtor()
-        {
-            Assert.Throws<ArgumentException>(() => new Bitmap((Image)null, Size.Empty));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapImageIntIntCtor()
-        {
-            Assert.Throws<ArgumentException>(() => new Bitmap((Image)null, int.MinValue, int.MaxValue));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapIntIntCtor()
-        {
-            Assert.Throws<ArgumentException>(() => new Bitmap(int.MinValue, int.MaxValue));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapIntIntGraphicCtor()
-        {
-            Assert.Throws<ArgumentNullException>(() => new Bitmap(1, 1, null));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapIntIntPixelFormatCtor()
-        {
-            Assert.Throws<ArgumentException>(() => new Bitmap(int.MinValue, int.MaxValue, PixelFormat.Format1bppIndexed));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapStreamCtor()
-        {
-            AssertExtensions.Throws<ArgumentNullException, ArgumentException>("stream", null, () => new Bitmap((Stream)null));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapStreamBoolCtor()
-        {
-            AssertExtensions.Throws<ArgumentNullException, ArgumentException>("stream", null, () => new Bitmap((Stream)null, true));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapStringCtor()
-        {
-            Assert.Throws<ArgumentNullException>(() => new Bitmap((string)null));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapStringBoolCtor()
-        {
-            Assert.Throws<ArgumentNullException>(() => new Bitmap((string)null, false));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapTypeStringCtor1()
-        {
-            Assert.Throws<NullReferenceException>(() => new Bitmap((Type)null, "mono"));
-        }
-
-        [ConditionalFact(Helpers.IsDrawingSupported)]
-        public void BitmapTypeStringCtor2()
-        {
-            AssertExtensions.Throws<ArgumentNullException, ArgumentException>("resource", null, () => new Bitmap(typeof(Bitmap), null));
-        }
```

Comment:
I understand why the NRE test is being deleted, but why the others?  Are these duplicates, and if so, of what?

Reply:
- `BitmapImageCtor`: dupe of `Ctor_NullImageWithoutSize_ThrowsNullReferenceException`
- `BitmapImageSizeCtor` and `BitmapImageIntIntCtor`: dupe of `Ctor_NullImageWithSize_ThrowsArgumentNullException`
- `BitmapIntIntGraphicCtor`: dupe of `Ctor_NullGraphics_ThrowsArgumentNullException`
- `BitmapIntIntPixelFormatCtor`: dupe of `Ctor_InvalidWidth_ThrowsArgumentException` and `Ctor_InvalidHeight_ThrowsArgumentException`
- `BitmapStreamCtor` and `BitmapStreamBoolCtor`: dupe of `Ctor_NullStream_ThrowsArgumentNullException`
- `BitmapStringCtor` and `BitmapTypeStringCtor2`: dupe of `Ctor_NullFilePath_ThrowsArgumentNullException`